students/21th/gunwookim/we/main.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <title>Main</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="stylesheet" href="./style/common.css" />
+    <link rel="stylesheet" href="./style/main.css" />
+  </head>
+  <body>
+    <nav>
+      <div class="logo-div">
+        <div class="logo-link-img">
+          <a href="#">
+            <img src="./images/instagram.png" alt="instagram" />
+          </a>
+        </div>
+        <div class="logo-link-span">
+          <a href="#">
+            <span>westagram</span>
+          </a>
+        </div>
+      </div>
+      <div class="search-div">
+        <div class="search-input">
+          <input type="text" class="searchInput" placeholder="검색" />
+        </div>
+        <div class="search-result" style="visibility: hidden"></div>
+      </div>
+      <div class="link-img-div">
+        <div class="explore-link-img">
```

Comment:
```suggestion
        <div class="explore-link-wrapper">
```
등의 className이 적합해보입니다 이 div tag가 img는 아니니까요